### ISSUE TITLE
Feature: Falldatenblätter für Textbausteine verwenden

### ISSUE THREAD

**Issue by iradraconis** [jlawyerorg/j-lawyer-forms#25]
Ich hatte so einen Gedanken, vielleicht auch nur als Frage an die Entwickler: wäre es technisch machbar oder sinnvoll, die Falldatenblätter als Baukasten für Schriftsätze zu benutzen? Ich stelle mir das so vor, dass es ein Falldatenblatt ähnlich dem bestehendem zum Sozialrecht gibt, in dem der User seine Textbausteine, die irgendwo zentral gespeichert werden (Server) auswählt mit fallspezifischen Daten versieht und dann einen Text generieren lässt, der dann in ein Dokument kopiert werden kann? Eine Art Schriftsatzgenerator? 

Wäre etwas in der Art mit den aktuellen Gegebenheiten technisch umsetzbar ?

**Comment by j-dimension** [comment]
Ja, absolut denkbar. Soetwas hatte ich tatsächlich in dem Onlinetraining zum Thema Falldatenblätter live umgesetzt, da ging es um eine Art Fluggastrecht "light", wo dann aus dem Falldatenblatt ein Schriftsatz (der Text dafür) erstellt wurde.

Müsste man an einem konkreten Beispiel diskutieren.

**Comment by iradraconis** [comment]
Es müsste einen einfach zu bedienenden Pool von Textbausteinen (Dateiformat?) geben, der dann angesteuert werden kann. Die Bausteine müssten im besten Fall jederzeit anpassbar sein. Vielleicht könnte sich auch jeder User selbst einen Pool anlegen.
